spec/models/dialog_field_drop_down_list_spec.rb

```diff
@@ -236,6 +236,35 @@
         end
       end
 
+      context "dialog field dropdown without options hash" do
+        before do
+          @df = FactoryGirl.create(:dialog_field_drop_down_list, :name => 'test drop down')
+        end
+
+        it "#multi_value?" do
+          expect(@df.multi_value?).to be_falsey
+
+          @df.force_multi_value = true
+          expect(@df.multi_value?).to be_truthy
+        end
+
+        it "#automate_key_name" do
+          expect(@df.automate_key_name).to eq("dialog_#{@df.name}")
+        end
+      end
+
+      context "dialog field dropdown with options hash" do
+        before do
+          @df = FactoryGirl.create(:dialog_field_drop_down_list,
+                                   :name    => 'test drop down',
+                                   :options => {:force_multi_value => true})
+        end
+
+        it "#multi_value?" do
+          expect(@df.multi_value?).to be_truthy
+        end
+      end
+
```

Comment:
As it stands, these tests are right above the highlighted lines, github just didn't pick up on the fact that I did include them per your comment. Can you take a look sometime please and tell me if it's okay?